gql/utils.go

```diff
@@ -9,18 +10,22 @@ import (
 
 // DeepExtractField utilizes ExtractField multiple times to retrieve the value of a field in a multilevel object.
 // The key is expected to use FieldPathSeparator to distinguish the multiple levels.
-func DeepExtractField(s interface{}, key string) interface{} {
+// This will throw an error if the value for the first split cannot be found.
+func DeepExtractField(s interface{}, key string) (interface{}, error) {
```

Comment:
Blame it on the guy who wrote this but this function is exported so changing it is a breaking change. I think it would be best to make an unexported version that does the work and turn the exported one into a function swallowing the error.

Reply:
Good thinking, will do